app/helpers/result_filters/subject_helper.rb

```diff
@@ -1,6 +1,7 @@
 module ResultFilters
   module SubjectHelper
     include FilterParameters
+    include CsharpRailsSubjectConversionHelper
```

Comment:
@davidgisbey - I think we need a tech debt card to remove the csharp subjecct id conversion - I can't imagine anyone would still be querying the site with legacy ids ...

Reply:
Great. I'll card it up 👍